modules/ROOT/pages/cloudhub-networking-guide.adoc

```diff
@@ -124,6 +124,17 @@ covers the following regions.
 
 |===
 
+https://anypoint.mulesoft.com[Anypoint Platform] for the MuleSoft Government Cloud
```

Comment:
covers the following regions:

Reply:
Agree with Priya.

Reply:
Left the "." to be consistent with other sections to minimize changes since I don't own this file.